Tests/Unit/RealMiniAppTests.swift

```diff
@@ -145,7 +150,7 @@ class RealMiniAppTests: QuickSpec {
                 it("will return valid Mini App View instance with a default hostAppMessageDelegate and getUniqueId() will return an error message") {
                     let responseString = """
                     [{
-                        "id": "123",
+                        "id": "app-id-test",
```

Comment:
can you refactor it with a class or global variable?

Reply:
Done